src/atomate2/vasp/sets/defect.py

```diff
@@ -46,13 +181,87 @@ def get_incar_updates(
             A dictionary of updates to apply.
         """
         return {
-            "IBRION": 2,
+            "ALGO": "All",
+            "IBRION": 1,
+            "LAECHG": False,
             "ISIF": 2,
+            "EDIFF": 1e-5,
             "EDIFFG": -0.05,
             "LREAL": False,
             "NSW": 99,
-            "LCHARG": False,
             "ENCUT": 500,
+            "NELMIN": 6,
```

Comment:
Need to set GGA = PE in this and other sets.

Reply:
added